src/04-accordion/style.scss

```diff
@@ -1,9 +1,138 @@
 @charset "UTF-8";
 /*********************************************************************************************
+
 Plugin Name: MOJ Blocks
-Author:  Ministry of Justice
-License: GNU General Public License v3 or later
-License URI: http://www.gnu.org/licenses/gpl-3.0.html
+Block: Accordion (frontend styles)
 Text Domain: mojblocks
 
 **********************************************************************************************/
+
+/** Variables */
+$mojblocks-color-light-blue: #1d70b8;
+$mojblocks-color-teal: #336C83;
+$mojblocks-color-dark-blue: #003078;
+$mojblocks-color-grey: #6F777B;
+$mojblocks-color-light-grey: #C7CDD0;
+
+// https://design-system.service.gov.uk/components/accordion/
+@import "node_modules/govuk-frontend/govuk/components/accordion/accordion";
+
+/*********************************************************************************************
+
+SASS modifications to conform GovUK frontent to MoJ block styling
+
+**********************************************************************************************/
+
+// Change postion and colour of title and arrow to align within section
+.js-enabled .govuk-accordion__section-header {
+    padding: 20px 28px 20px 10px;
+    color: $mojblocks-color-light-blue;
+}
+
+.govuk-accordion__section-heading {
+    margin-left: 22px;
+}
+
+// Change positon of open/close icon to far left side
+.js-enabled .govuk-accordion__icon {
+    right: 0;
+    left: -30px;
+    width: 16px;
+    height: 16px;
+    margin-top: -8px;
+}
+
+// Remove "mustaches" underline caused by NHS styles conflicting with Gov UK
+.govuk-accordion__section-button,
+.govuk-accordion__open-all {
+    box-shadow: none;
+}
+
+// Add a bottom border to accordion sections (on last one)
+.js-enabled .govuk-accordion {
+    border-bottom: none;
+
+    &:last-child {
+        border-bottom: 1px solid $mojblocks-color-light-grey;
+    }
+}
+
+// Accordion title - stop NHS theme causing "link bounce" effect when clicked on
+.js-enabled .govuk-accordion__section-button:active,
+button:active {
+    top: 0;
+}
+
+// Open/close button - stop NHS theme causing "link bounce" effect when clicked on
+.js-enabled .govuk-accordion__open-all:active,
+button:active {
+    top: 0;
+}
+
+// Reset NHS margin to display no margin on accordion sections
+.govuk-accordion {
+    margin-bottom: 0;
+}
+
+// Replace Gov plus symbol with arrow symbol (when closed)
+.js-enabled .govuk-accordion__icon::before {
+    content: "";
```

Comment:
`::` isn't accepted in all browsers for :before - maybe this is the reason for your bug?

Reply:
Thanks I'll investigate!